src/core/Akka/Dispatch/Mailboxes.cs

```diff
@@ -277,38 +277,36 @@ public MailboxType GetMailboxType(Props props, Config dispatcherConfig)
                 _mailboxSizeWarningIssued = true;
             }
 
-            Func<MailboxType, MailboxType> verifyRequirements = mailboxType =>
+            MailboxType VerifyRequirements(MailboxType mailboxType)
```

Comment:
This might be interesting - if the `VerifyRequirements` local function is able to be created using a value type, which the compiler tries to do by default, this might improve the rate at which we're able to spawn actors. Will check benchmark results to verify.